.gitignore

```diff
@@ -0,0 +1,13 @@
+**/*.log
+**/node_modules
+.coverage
+.nyc_output
+docs/
+out/
+build/
+system-test/secrets.js
+system-test/*key.json
+*.lock
+.DS_Store
+google-cloud-logging-winston-*.tgz
+google-cloud-logging-bunyan-*.tgz
```

Comment:
add a new line pls :)

Reply:
It's not here, it's in repo-tools. But I agree it should be added :)

Reply:
https://github.com/GoogleCloudPlatform/nodejs-repo-tools/issues/129

Reply:
And https://github.com/GoogleCloudPlatform/nodejs-repo-tools/pull/130 as well. It's a quick fix.